libs/designsystem/src/lib/components/segmented-control/segmented-control.component.scss

```diff
@@ -64,7 +62,26 @@ ion-segment-button {
   --margin-end: 0;
   --margin-start: 0;
   --margin-top: 0;
+
+  min-height: utils.size('xl');
+  font-weight: utils.font-weight('normal');
+  font-size: utils.font-size('s');
+  text-transform: none;
   margin: 0;
+
+  // NOTE: Also see ion-back-button in page component
```

Comment:
Same point as comment regarding references to page and segmented control component.

Reply:
Same answer as above 🙂